adoc/deployment-azure.adoc

```diff
@@ -0,0 +1,217 @@
+== Deployment on Microsoft Azure
+
+Deployment on Microsoft Azure is currently a tech preview.
+
+.Preparation Required
+[NOTE]
+====
+You must have completed <<deployment.preparations>> to proceed.
+====
+
+You will use {tf} to deploy the whole infrastructure described in
+<<architecture-azure>>. Then you will use the `skuba` tool to bootstrap the
+{kube} cluster on top of it.
+
+
+[[architecture-azure]]
+=== AZURE Deployment
+
+The AZURE deployment created by our {tf} template files leads to the
+creation of the infrastructure described in the next paragraphs.
+
+==== Setup service principal for terraform login credential
+
+Following the guide link:https://www.terraform.io/docs/providers/azurerm/guides/service_principal_client_secret.html#creating-a-service-principal-in-the-azure-portal[Creating a Service Principal in the Azure Portal], and set up `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, `ARM_SUBSCRIPTION_ID`, `and ARM_TENANT_ID` in `container-openrc.sh`.  Source `container-openrc.sh` before deploying terraform script.
```

Comment:
Maybe could make the credential part more consistent with this PR -> https://github.com/SUSE/doc-caasp/pull/903/files

Reply:
changed